### PR TITLE
improvement(tools): optimize convert-pth-to-ggml

### PR DESCRIPTION
**Pull request by tpoisonooo**
optimize convert tool with `argparse`

```python
$ python3 convert-pth-to-ggml.py -h
usage: convert-pth-to-ggml.py [-h] dir_model {f32,f16} out_dir

Convert ckpt models to ggml models. For example: python3 convert-pth-to-ggml.py ../llama-models/7B/ f32 models/llama-7B

positional arguments:
  dir_model   Directory path of the checkpoint model
  {f32,f16}   Data type of the converted tensor, f32 or f16
  out_dir     Directory path for storing ggml model

options:
  -h, --help  show this help message and exit
```

Tested on 7B/30B models, it works well.
```bash
$ tree models/
models/
├── 7B
├── llama-30B
│   ├── ggml-model-f16.bin
│   ├── ggml-model-f16.bin.1
│   ├── ggml-model-f16.bin.2
│   └── ggml-model-f16.bin.3
└── llama-7B
    ├── ggml-model.bin -> ggml-model-f16.bin
    ├── ggml-model-f16.bin
    └── ggml-model-f32.bin
```

### CODE REVIEW

**Comment by tpoisonooo** [comment]
cc @ggerganov

**Comment by tpoisonooo** [comment]
Conflict fixed and tested on 7B/30B. 

master version diffs with 3 lines [here](https://github.com/ggerganov/llama.cpp/commit/2af23d30434a677c6416812eea52ccc0af65119c) :
```python
if os.path.exists(fname_out):
    print(f"Skip conversion, it already exists: {fname_out}")
    sys.exit(0)
```


 cc @gjmulder

File: convert-pth-to-ggml.py [hunk 1]
```diff
@@ -22,19 +22,29 @@
 import struct
 import numpy as np
 import torch
+import argparse
+import os
```
Comment (sw) [inline convert-pth-to-ggml.py:26]: `os` is already imported

**Comment by sw** [comment]
can you please update the readme? the choice "1" for ftype will no longer work.

**Comment by sw** [comment]
This is almost a duplicate of #109

**Comment by ggerganov** [comment]
Decided to go with #109

**Pull request closed without merging**